.github/workflows/continuous-integration.yml

```diff
@@ -19,6 +19,8 @@ jobs:
             target: x86_64-unknown-linux-musl
           - os: macos-latest
             target: x86_64-apple-darwin
+          - os: macos-latest
+            target: aarch64-apple-darwin
```

Comment:
maybe something like
```yaml
matrix:
  build_name: [windows, ubuntu, macos-x86, macos-darwin]
    include:
      - build_name: windows
        os: windows-latest
        target: x86_64-pc-windows-msvc
      - build_name: ubuntu
      # ...
```

Reply:
similar solution for build-for-release

Reply:
my point is that github's actions will not run 4 separate builds but only 3, because of how matrix works. you need 4 items in the thing outside of `include`